draft-ietf-mls-protocol.md

```diff
@@ -2076,21 +2095,28 @@ A member of the group applies a Commit message by taking the following steps:
   applied at the leftmost unoccupied leaf, or appended to the right edge of the
   tree if all leaves are occupied.
 
-* Process the `path` value using the ratchet tree the provisional GroupContext,
-  to update the ratchet tree and generate the `commit_secret`:
+* Verify that the `path` value is populated if either of the `updates` or
```

Comment:
I think we could be more flexible here: Since the path value MAY be omitted if the Commit only contains Adds, it MAY also not be omitted.
The sender should be able to choose whether they want to use the commit to also do an update and the receivers should simply process it.

Reply:
I think this is already the case, but I added a bit more text to clarify